### ISSUE TITLE
[regression] The titlebar can't be hidden anymore since sway 1.8

### ISSUE THREAD

**Issue by kit-ty-kate** [swaywm/sway#7350]
- **Sway Version:** 1.8

The following configuration mentioned in https://github.com/swaywm/sway/issues/6946, https://github.com/swaywm/sway/issues/5050 and https://github.com/swaywm/sway/issues/1286:
```
font pango:monospace 0
titlebar_border_thickness 0
titlebar_padding 0
```
Worked just fine in Sway 1.7 but doesn't work anymore in 1.8 due to https://github.com/swaywm/sway/pull/7092

As a temporary workaround `font pango:monospace 1` can be used instead of `font pango:monospace 0` but it's not the best solution.

**Comment by emersion** [comment]
Hiding the titlebar this way was a hack to begin with.

**Comment by kit-ty-kate** [comment]
That may be the case but when a hack is used by most people* i feel like it becomes a feature of its own.
I've personally never seen any sway config or screenshot where the user kept the titlebar.

*: arguably my sample set might be biased, but still.

**Comment by sweenu** [comment]
I remember when I first started sway loosing a lot of time trying to find the option to disable titlebars. It seemed obvious to me that such an option would exist.
There will also inevitably be lost time in supporting people like me that try to do that and giving them hacks.
I know sway isn't looking to add more options, but it seems to me that this would be worth the time.

If not, then I'm sadly in favor of reinstigating a 0 font size. Better than nothing.

**Comment by Inginn** [comment]
yes just broke my settup.
I dont want a bar.
why would I go minimalizt/maximize my screen usage and then add a bar I dont want.

**Comment by nicgene** [comment]
Same here. My current workaround is `font pango: monospace 0.001`

**Comment by antioxx** [comment]
> Same here. My current workaround is `font pango: monospace 0.001`
Thanks <3

**Comment by quackers19** [comment]
> Same here. My current workaround is `font pango: monospace 0.001`

**Comment by billop** [comment]
I believe the official way of hiding titlebars is setting `default_border` to `pixel [<n>]` or `none`.

**Comment by Phylex** [comment]
Yea just updated to 1.8 and have the same problem. I‘d really appreciate either having a setting for this or reinstating the fix.
Either way, sway has been a really nice experience and I‘d like to say thanks to all involved.

**Comment by mythi** [comment]
Is it related that with 1.8 and Chrome PWA apps I started getting *two* titlebars?

**Comment by emersion** [comment]
#7408 has been merged to the 1.8 branch as a temporary fix. The main branch still doesn't allow a zero font size, and this is tracked by #7409. Patches welcome to fix this.

**Comment by emersion** [comment]
Reminder that a new Sway release is upcoming and this feature will go away unless someone steps up.
